source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,276 @@
+   :copyable: false
+
+   [{_id ObjectID("...")} {title Captain America: The Winter Soldier} {year 2014}]
+   [{_id ObjectID("...")} {title Captain America: Civil War} {year 2016}]
+
+Sort by Relevance
+~~~~~~~~~~~~~~~~~
+
+A text search assigns a numerical text score to indicate how closely
+each result matches the string in your query filter. To reveal the text
+score in your output, use a projection to retrieve the "textScore"
+metadata. You can sort the text score in descending order by specifying
+a sort on the "textScore" metadata.
+
+Example
+```````
+
+The following example performs the following actions:
+
+- Runs a text search for titles that contain the term "Avenger"
+- Sorts the results in descending order based on their text score
+- Includes the ``title`` and ``score`` fields from the results
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/textSearch.go
+   :language: go
+   :dedent:
+   :start-after: begin text score
+   :end-before: end text score
+
+After running this example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [{title The Avengers} {score 1}]
+   [{title Avengers: Infinity War} {score 0.6666666666666666}]
+   [{title Captain America: The First Avenger} {score 0.625}]
+
+Aggregation
+-----------
+
+You can also include the ``$text`` evaluation query operator in the
+:manual:`$match </reference/operator/aggregation/match/>` stage to
+perform a text search in an aggregation pipeline.
+
+Term Search Example
+~~~~~~~~~~~~~~~~~~~
+
+The following example runs a text search for titles that contain the term "Winter":
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/textSearch.go
+   :language: go
+   :dedent:
+   :start-after: begin aggregate text search
+   :end-before: end aggregate text search
+
+After running this example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [{_id ObjectID("...")} {title Captain America: The Winter Soldier} {year 2014}]
+
+Text Score Example
+~~~~~~~~~~~~~~~~~~
+
+The following example performs the following actions:
+
+- Runs a text search for titles that contain the term "Avenger"
+- Sorts the results in descending order based on their text score
+- Includes the ``title`` and ``score`` fields from the results
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/textSearch.go
+   :language: go
+   :dedent:
+   :start-after: begin aggregate text score
+   :end-before: end aggregate text score
+
+After running this example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [{title The Avengers} {score 1}]
+   [{title Avengers: Infinity War} {score 0.6666666666666666}]
+   [{title Captain America: The First Avenger} {score 0.625}]
+
+Additional Information
+----------------------
+
+For more information on the operations mentioned, see the following
+guides:
+
+- :doc:`Specify a Query </fundamentals/crud/query-document>`
```

Comment:
Should this be:

```suggestion
- :doc:`filter </fundamentals/crud/query-document>`
```

Reply:
Suggest rephrasing the link text to be consistent with the last two bullet items